### ISSUE TITLE
Edit profile: Status message of saved changes not announced as soon as it appears and focus is not in correct order after save

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#42]
URL: https://hcdev.openplus.ca/en/user/203/edit
User Agent: Chrome, Firefox
Language: both

Issue: When the user makes changes on the page and hits enter at a field OR enters/clicks Save button, the status message is not read right away. 

Situation 1: The page gets saved but the focus stays around where the user was and screen reader continues from there. 
For example, if the user hits enter at Username and hits enter key after editing, the focus goes to Status field (3 fields below it).
If the user hits enter at First Name, the focus goes to Language settings (2 tabs below it).
If the user hits enter at Save, the focus goes to either Contact us section or stays on Save button.
![image](https://user-images.githubusercontent.com/14280062/128418057-4ff1ada1-24a8-4f4d-84ee-20a5c80ea7b2.png)

Situation 2: When there is an error message, the message does get announced but then the focus goes to the items right after where the error was. For example, when the Passwords do not match, the screen reader announced the status error message and then jumped to right after the Confirm password field.
![image](https://user-images.githubusercontent.com/14280062/128418082-8c50c57d-8762-444f-84c1-29404398a8dd.png)

Expected result: The status message should be announced and the focus should stay there. User can go to the error messages from there if there are errors.

**Comment by davidgotlieb** [comment]
Could not duplicate, likely fixed with broader WxT update

**Comment by hydegeoff** [comment]
The issue still exists.

I am using Chrome and NVDA ver. 2021.2 

Steps to reproduce 
1. Login into https://rsams.dev.cloud.openplus.ca/en/user/
2. Go to the edit profile tab
3. Turn on NVDA
4. Make a small change for instance change the your language 
5. Select the Save button 
6. Once the page loads again, stop moving the cursor/mouse, listen and wait until the screen reader stops talking.  It should have announce the text in status message, but it doesn't. 
![Status message does not get announced by SR](https://user-images.githubusercontent.com/88147013/161107242-d9a3f62e-5f7a-49d7-99c1-8935c1a585ff.jpg)


Expected Result: 
Screen readers should immediately announce all status messages upon performing an action.

**Comment by echouin** [comment]
No longer an issue.
